docs/dev/wrapping.rst

```diff
@@ -59,7 +59,8 @@ The cosmology class looks something like this:
 Now let's wrap this library.  First, we need to create a wrapping of the
 top-level namespace so that it implements the
 :class:`cosmology.api.CosmologyNamespace` interface. This will also require
-creating a namespace for the constants.
+creating a namespace for the constants. We don't recommend building dynamic
+libraries and modules and do it here only for demonstration purposes.
```

Comment:
Let's not demonstrate it if we don't recommend it. Every code will be copied and pasted blindly, without looking at the caveat.

Reply:
How can we show making a new python file with these contents?
Esp, I'd like to doctest these so the examples always remain correct.

Reply:
Maybe we could just show the source layout and files of the wrapper explicitly?

```py
# wrapper/__init__.py
from . import constants
...
```
```py
# wrapper/constants.py
from ... import ...
...
```

Reply:
Sure! I think I have a way to do this where we show something good but invisibly hack about with the SimpleNamespace for the doctests... working on it now.

Reply:
Done. Now we have doctests with https://sybil.readthedocs.io/en/latest/rest.html#restructured-text-parsers